.travis.yml

```diff
@@ -9,16 +9,19 @@ jobs:
       stage: test
       compiler: clang
       python: 2.7
+      node_js: 7
       addons:
         apt:
           sources: ['ubuntu-toolchain-r-test']
-          packages: ['cmake', 'nodejs', 'g++-5']
+          packages: ['cmake', 'g++-5']
       before_install:
         - export CC="${CC_COMPILER}"
         - export CXX="${CXX_COMPILER}"
         - export ASAN_OPTIONS="symbolize=1"
       install:
         - pip install --user flake8
+        - "curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -"
```

Comment:
I tried the polyfill, and it worked, but old node has another error preventing us from using it:
```
function check272() {                                                                                                                                                                                                                                                                     
  console.error(2147483647|0, 2147483648|0, asmModule.le_s(2147483647|0, 2147483648|0))                                                                                                                                                                                                           
  return (asmModule.le_s(2147483647 | 0, 2147483648 | 0) | 0 | 0) == (0 | 0) | 0;                                                                                                                                                                                                         
}                                                                                                                                                                                                                                                                                         
                                                                                                                                                                                                                                                                                          
if (!check272()) fail272();
```
This is one of the generated test cases. I added the console.error call. It prints `2147483647 -2147483648 1` and fails. The function implementing `asmModule.le_s` is
```
 function $$23(x, y) {
  x = x | 0;
  y = y | 0;
  return (x | 0) <= (y | 0) | 0;
 }
```
which I believe is correct.